CommunicationLayer/MessagingFramework/BatteryDataMessage.cpp

```diff
@@ -28,29 +50,11 @@ float BatteryDataMessage::batteryCurrent() const
    return getFloat(messageData_, BATTERY_CURRENT_INDEX);
 }
 
-float BatteryDataMessage::stateOfCharge() const
```

Comment:
I think removing these fields from the BatteryDataMessage is misleading. They are still there in the protocol and could lead to confusion as the BatteryDataMessage is no longer fulfilling the protocol. So unless it is removed from the CCS and here (and hopefully update the protocol document) then I would keep it in.